File: 0926. Flip String to Monotone Increasing/Solution.py
class Solution:
    def minFlipsMonoIncr(self, s: str) -> int:
        cnt_ones = cnt_flips = 0
        for c in s:
            if c == '1':
                cnt_ones += 1
            else:
                cnt_flips = min(cnt_flips + 1, cnt_ones)
        return cnt_flips

File: 0926. Flip String to Monotone Increasing/Solution.c
#include <math.h>

int minFlipsMonoIncr(char* s)
{
    int cnt_ones = 0, cnt_flips = 0;
    while (*s) {
        if (*s++ == '1')
            ++cnt_ones;
        else
            cnt_flips = fmin(cnt_flips + 1, cnt_ones);
    }
    return cnt_flips;
}
